ksy_schema.json

```diff
@@ -216,7 +224,29 @@
             }
           }
         },
-        "license": { "type": "string" },
+        "tags": {
+          "type": "array",
+          "description": "list of some identifiers that can give additional information about the format\n\nshould be written in `lowercase-kebab-case` and listed in alphabetical order\n\nshould be used only at the top level",
```

Comment:
I'm pondering over this one:

> **should be written in `lowercase-kebab-case`** and listed in alphabetical order

The key `meta/tags` is currently used on [the KSF site](https://formats.kaitai.io/) - a KSY spec is included into a category according to the folder in which is it located **or** if it has the corresponding `tag` matching the name of the folder. We opted for `snake_case` for the names of all .ksy files and directories ([`machine_code`](https://github.com/kaitai-io/kaitai_struct_formats/tree/a8b9cd0948fff5ab2879af2d93dc7a0683a5721b/machine_code)), so I suppose `meta/tags` should also use it?

If you create a format with this:

```ksy
meta:
  tags:
    - machine_code
```

it will be added into the _🏭 CPU / Machine Code Disassembly_ category at [formats.kaitai.io](https://formats.kaitai.io/).

Reply:
Maybe. There was no examples of multi-word tags, so I assumed that kebab-case would be convenient.